doc/sphinx/practical-tools/coqide.rst

```diff
@@ -327,3 +327,123 @@ If you choose something else than UTF-8, then missing characters will
 be written encoded by `\x{....}` or `\x{........}` where each dot is
 an hexadecimal digit: the number between braces is the hexadecimal
 Unicode index for the missing character.
+
+.. _coqide-debugger:
+
+Debugger
+--------
+
+Version 8.14 introduces a preliminary visual debugger for Ltac tactics within
+CoqIDE.  It supports setting breakpoints visually, automatically
+displaying the stopping point in the source code with "continue",
+"step over" "step in" and "step out" operations.  It does not yet
+display the call stack or variable values.
+
+The debugger is based on the :ref:`Ltac debugger <interactive-debugger>`.
```

Comment:
```suggestion
The debugger is based on the :ref:`|Ltac| debugger <interactive-debugger>`.
```

Reply:
I think the use `|Ltac|` will likely go away at some point.  In on-line messages, no one bothers to use subscripts.  I'm not so keen on the look, either.